src/main/java/com/khutircraftubackend/seller/SellerRepository.java

```diff
@@ -9,5 +9,7 @@
  */
 
 @Repository
-public interface SellerRepository extends JpaRepository<Seller, Long> {
+public interface SellerRepository extends JpaRepository<SellerEntity, Long> {
+
+    SellerEntity findByUser(UserEntity user);
```

Comment:
do we want to ignore `id` field when mapping to SellerEntity? 
coz SellerDTO doesn't have it 

```
@Mapping(target = "id", ignore = true)
```

Reply:
setting up seller classes is not included in this task. registration only.